### ISSUE TITLE
question about the loss

### ISSUE THREAD

**Issue by harlem867** [xuannianz/keras-GaussianYOLOv3#6]
@xuannianz  hi, thank you for this good project!
I have noticed that in your implementation, the gaussian loss is 
    _pi = tf.constant(np.pi)
    Z = (2 * pi * (sigma + sigma_const) ** 2) ** 0.5
    probability_density = tf.exp(-0.5 * (x - mu) ** 2 / ((sigma + sigma_const) ** 2)) / Z
    nll = -tf.log(probability_density + 1e-7)_
and want to know how the mu and sigma will change during the training.
by my inference， maybe the mu should approach to the true value of x,y,w,h, and the sigmas should be reduced to 0.  Is this right？

**Comment by xuannianz** [comment]
Absolutely.

**Comment by harlem867** [comment]
According to the gaussian loss fuction of x,y,w,h, it can be a negative value.
How much training loss can be reduced during the traning using your gaussian loss？（sigma_const=0.3）

**Comment by xuannianz** [comment]
Absolutely.

**Comment by xuannianz** [comment]
https://github.com/jwchoi384/Gaussian_YOLOv3/issues/27
https://github.com/jwchoi384/Gaussian_YOLOv3/issues/19
FYI.
